README.md

```diff
@@ -37,12 +40,44 @@ bundles/ezsystemsprivacycookie/css/privacycookie.css
 bundles/ezsystemsprivacycookie/js/privacycookie.js
 ```
 
+Add the following minimal configuration in `config.yml` file to enable `Assetic` support in your application (Symfony 2.8 and later):
+
+```yaml
+assetic:
+    debug: '%kernel.debug%'
+    use_controller: '%kernel.debug%'
+    filters:
+        cssrewrite: ~
+```
+
 If you are installing bundle via `composer require` you must also copy assets to your project `web` directory. You can do this by calling Symfony built-in command from the project root directory:
```

Comment:
If you are installing the bundle via `composer require` you must also copy assets to your project's `web` directory. You can do this by calling Symfony's built-in command from the project root directory: